src/app/tests/suites/certification/Test_TC_RVCCLEANM_1_1.yaml

```diff
@@ -42,25 +42,13 @@ tests:
               type: int16u
 
     - label: "Step 3: TH reads from the DUT the FeatureMap attribute."
-      PICS: " !RVCCLEANM.S.F00 "
       command: "readAttribute"
       attribute: "FeatureMap"
       response:
           value: 0
           constraints:
               type: bitmap32
 
-    - label:
-          "Step 3: Given RVCCLEANM.S.F00(DEPONOFF) ensure featuremap has the
-          correct bit set"
-      PICS: RVCCLEANM.S.F00
-      command: "readAttribute"
-      attribute: "FeatureMap"
-      response:
-          constraints:
-              type: bitmap32
-              hasMasksSet: [0x1]
-
```

Comment:
Step 4c, below, which looks at RVCCLEANM.S.F00 (not the negated one, which, confusingly, is also called 4c), should also be removed.